shepherd/storage/minio_storage.py

```diff
@@ -1,10 +1,34 @@
+import os
+
+from os import path
+
+import logging
 from io import BytesIO
 
 from minio import Minio
+from minio.error import MinioError
+from urllib3.exceptions import HTTPError
+
+from ..errors.api import StorageError, StorageInaccessibleError
+from ..constants import DONE_FILE, ERROR_FILE, INPUT_DIR, OUTPUT_DIR
+from .storage import Storage
+from ..utils import minio_object_exists
 
-from shepherd.constants import DONE_FILE, ERROR_FILE
-from shepherd.storage.storage import Storage
-from shepherd.utils import minio_object_exists, pull_minio_bucket, push_minio_bucket
+
+def inherit_doc(parent):
```

Comment:
I am sorry for not suggesting it earlier but I would really prefer a simple `Implementing [sphinx link]` docsting over this...